### PR TITLE
KOGITO-1842 adding endpoint link to process details page

### PR DESCRIPTION
**Pull request by AjayJagan**
- added link support for endpoint in process details page

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@AjayJagan this needs rebase

File: packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx [hunk 1]
```diff
@@ -108,9 +72,18 @@ const ProcessDetails: React.FC<IOwnProps> = ({ data, from }) => {
             </Text>
           </FormGroup>
           <FormGroup label="Endpoint" fieldId="endpoint">
-            {data.ProcessInstances[0].endpoint ? (
+            {data.ProcessInstances[0].serviceUrl ? (
```
Comment (cristianonicolai) [inline packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx]: @AjayJagan should we reuse `EndpointLink` from https://github.com/kiegroup/kogito-apps/pull/177 in here?

Comment (srambach) [reply packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx]: I agree, I think this should be consistent between the two and use the same component. If it doesn't make sense to display the URL on the listing, then don't do it here either. If it does, then put it on the listing as well.

File: packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx [hunk 2]
```diff
@@ -108,9 +72,18 @@ const ProcessDetails: React.FC<IOwnProps> = ({ data, from }) => {
...
               <Text component={TextVariants.p}>
-                {data.ProcessInstances[0].endpoint}
+                <Button
```
Comment (srambach) [inline packages/management-console/src/components/Organisms/ProcessDetails/ProcessDetails.tsx]: Button will give this `white-space:nowrap`. So I guess we need to set `white-space:normal` here.
And to make a long URL, and anything else (like a long id) wrap, then we also need to set `word-break:break-all`. I think this is best done maybe on the `form` tag on line 51.
Ideally, we should have a top level CSS (or SCSS) file to place some utility-type classes. If we don't want to do that in this PR, let's make an issue and add the styles inline here for the moment. You can do it inline like this: 
```
<Form style={{
                // @ts-ignore
                'word-break': 'break-all'}}>
```

**Review by nmirasch — APPROVED** [review]

**Review by srambach — APPROVED** [review]
I created KOGITO-1967 to add a utility class for wrapping long, unbroken text like the URL. So this is good to go. 👍

**Review by cristianonicolai — APPROVED** [review]

**Comment by cristianonicolai** [comment]
@Sara4994 looks ok to you?

**Comment by cristianonicolai** [comment]
@AjayJagan seems like you need to update the snapshot tests

**Review by Sara4994 — APPROVED** [review]

**Comment by Sara4994** [comment]
@cristianonicolai @AjayJagan Yep, looks good to me

**Pull request merged**